Dockerfile.goreleaser

```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.15
+
+ARG CONFIG
+ENV CONFIG=$CONFIG
+
```

Comment:
We dont need complicate with config if we provide seperate dockerfile

Reply:
This doesn't include the config file in the docker. Should we include it?